TestCLRT.py

```diff
@@ -0,0 +1,19 @@
+import sys
```

Comment:
We will use only one Tester.py file. Kindly test in that only.
Don't create new  tester files.